atom/browser/api/atom_api_download_item.cc

```diff
@@ -111,7 +117,6 @@ bool DownloadItem::CanResume() const {
 
 void DownloadItem::Cancel() {
   download_item_->Cancel(true);
-  download_item_->Remove();
```

Comment:
why doesn't this need to be called anymore?

Reply:
`Remove` should only be called when we want to remove it from the download list